userguide/tutorials/getting_started.adoc

```diff
@@ -136,47 +136,55 @@ You can also go to http://<IP>:8080/api.html to explore the KillBill APIs.
 
 === Tomcat
 
-Users familiar with Java technologies can also install Kill Bill directly in their Web container of choice, such as Tomcat. We recommend using Tomcat version 8.5.
+Users familiar with Java technologies can also install Kill Bill directly in the Web container of their choice, such as Tomcat. We recommend using Tomcat version 8.5.
 
-If you don't have it yet, you can download the Tomcat container http://tomcat.apache.org/download-80.cgi[here] (get the Core binary distribution). The documentation is available http://tomcat.apache.org/tomcat-8.0-doc/index.html[here].
+==== Tomcat Installation
 
-In the rest of this guide, we will assume you have installed Tomcat in a directory called *CATALINA_HOME*.
+. Download the Tomcat container from http://tomcat.apache.org/download-80.cgi[here] (get the Core binary distribution). 
 
-Notes:
+. Install Tomcat as per the instructions specified http://tomcat.apache.org/tomcat-8.0-doc/index.html[here].
+
+. The rest of the guide assumes that you have installed Tomcat in a directory called *CATALINA_HOME*.
```

Comment:
Should we refer to $CATALINA_HOME (as an env variable and based on previous instructions provide an example value?)

Reply:
$CATALINA_HOME is not required as an environment variable. I have now changed it to TOMCAT_HOME in order to be able to refer to this path further on in the document.